flink-end-to-end-tests/flink-sql-client-test/pom.xml

```diff
@@ -91,6 +91,7 @@ under the License.
 				<artifactId>maven-shade-plugin</artifactId>
 				<executions>
 					<execution>
+						<id>e2e-dependencies</id>
```

Comment:
This ID was added so that this jar is excluded from the optional checks. It is implicitly ignored since the shade parser only parses certain executions.